lib/tracesConverter.js

```diff
@@ -102,6 +122,158 @@ var getValue = function (statement) {
 
 /**
  *
+ * Given an xAPI statements returns the
+ * 'type' value extracted from the 'object.definition' field.
+ *
+ * @param statement xAPI statement
+ */
+var getType = function (statement) {
+    var object = statement.object;
+    if (object) {
+        var definition = object.definition;
+        if (definition) {
+            var definitionType = definition.type;
+            if (definitionType) {
+                var index = definitionType.lastIndexOf('/');
+                if (index !== -1) {
+                    return definitionType.substring(index + 1);
+                }
+            }
+        }
+    }
+    return null;
+};
+
+/**
+ *
+ * Given an xAPI statements returns the 'response' value extracted from the
+ * 'result.response' field.
+ *
+ * @param statement xAPI statement
+ */
+var getResponse = function (statement) {
+    var result = statement.result;
+    if (result) {
+        return result.response;
+    }
+    return null;
+};
+
+/**
+ *
+ * Given an xAPI statement returns the
+ * 'event' value extracted from the 'verb.id' field.
+ *
+ * @param statement xAPI statement
+ */
+var getEvent = function (statement) {
+    var verb = statement.verb;
+    if (verb) {
+        var verbId = verb.id;
+        if (verbId) {
+            var index = verbId.lastIndexOf('/');
+            if (index !== -1) {
+                return verbId.substring(index + 1);
+            }
+        }
+    }
+    return null;
+};
+
+/**
+ *
+ * Given an xAPI statement returns the
+ * 'timestamp' value extracted from the 'statement.timestamp' field.
+ *
+ * @param statement xAPI statement
+ */
+var getTimeStamp = function (statement) {
+    return statement.timestamp;
+};
+
+/**
+ * Receives an xAPI Profile Statement and returns
+ * the equivalent data in Realtime format.
+ *
+ * xAPI Profile Statement info:
+ *
+ *     - https://github.com/e-ucm/tracking-model/blob/master/Tracking%20Model.md
+ *     - https://docs.google.com/spreadsheets/d/1o1qukRVI_eWpgnarh3n506HbzT1QTxerJ9eIfOfybZk/edit#gid=0
+ *
+ * Supported xAPI Statements are:
+ *      - Completable
+ *      - Variable
+ *      - Alternative
+ *      - Reachable
+ *
+ * Realtime Format info:
+ *
+ *      Composed by a  JSON Object that contains the basic information of the
+ *      xAPI Profile Statement in the following format:
+ *
+ *      {
+ *          event: <the verb.id, e.g. https://w3id.org/xapi/seriousgames/verbs/set --> 'set'>,
+ *          target: <the object.id, e.g. https://w3id.org/xapi/seriousgames/activities/variable --> 'variable'>,
+ *
+ *          value: <the result.extensions.value, e.g. http://rage-eu.com/xapi/extensions/value: 1 --> '1'>,
```

Comment:
Sorry, i don't understand the meaning of X ---> 'X' in http://rage-eu.com/xapi/extensions/value: 1 --> '1'. Can you explain better what are you trying to say?